virttest/qemu_monitor.py

```diff
@@ -1576,16 +1576,20 @@ def _get_migrate_parameter(self, parameter, error_on_missing=False,
         return super(HumanMonitor, self)._get_migrate_parameter(
             parameter, error_on_missing, disable_auto_x_evaluation)
 
-    def set_migrate_parameter(self, parameter, value, error_on_missing=False):
+    def set_migrate_parameter(self, parameter, value, error_on_missing=False,
+                              disable_auto_x_evaluation=True):
         """
         Set parameters of migrate.
 
         :param parameter: the parameter which need to set
         :param value: the value of parameter
+        :param disable_auto_x_evaluation: Whether to automatically choose
+                                          param with/without "x-" prefix
```

Comment:
I would just spell 'parameters' instead of 'param'. Besides that, the change looks good. What do you think?